openapi.yml

```diff
@@ -124,6 +124,25 @@ paths:
           example: 'druid:bc123df4567.json'
           schema:
             $ref: '#/components/schemas/DruidWithOptionalFormat'
+  /v1/objects/{id}/validate_moab:
+    get:
+      tags:
+        - objects
+      summary: Starts a validate moab job, which does a validation of the on disk Moab, without reference to the preservation-catalog basis
+      responses:
+        '200':
+          description: Job queued
+        '400':
+          description: Bad request (e.g., malformed druid)
+        '404':
+          description: Object not found
```

Comment:
i'd be inclined to drop the 404, as i don't think this will ever be a response from the current controller method -- the controller method doesn't check for presence itself, and the job can't report that back to the caller since the job is run async.

if no dir can be found for the Moab, the job'll report failure and then a person can retry the workflow step (once that's all wired up by subsequent tickets).